src/threads/hasThread.ts

```diff
@@ -0,0 +1,26 @@
+import type { APIGatewayProxyHandler } from 'aws-lambda';
+import { dynamodb } from '../dynamodb';
+import { badRequest, ok, serverError, unauthorized } from '../utils/responses';
+import { send } from './messages';
+import { extractTokenData } from '../utils/parsers';
+import { ForumTable } from './table';
+
+const forumTable = new ForumTable(dynamodb);
+
```

Comment:
I am clearly missing a spec somewhere which states what the "service" is supposed to do, with what input, what it sends, side effects ... The only spec should not be my initial scratch spec. I don't where it should be but we need to do it. 
Without that, just validating the tests is "difficult".

Reply:
> Should I host the generated documentation using the serverless config file? 

No, I think will deploy it statically, I am not sure I will use serverless tool for deployment actually.